### DISCUSSION TITLE
Specific Logging for multi-node cluster

### DISCUSSION THREAD

**Discussion by mnoumannawaz** [nats-io/nats-server#6095] [General]
We have a 3-node NATS cluster and want to enable logging specifically to capture critical events, such as node disconnections, cluster breaks, or service dis-connectivity, without enabling excessive debug logs.
Which log level or specific settings should we use to capture these types of events effectively?
Are there best practices for configuring NATS logs for focused cluster health monitoring?

**Comment by derekcollison** [comment]
From the system account listen on `"$SYS.ACCOUNT.*.DISCONNECT`

**Reply by mnoumannawaz** [reply]
System account is not accessible at this point might try this when I get access, I have monitoring port enabled Can I check somehow anything related to this over there ?

**Reply by derekcollison** [reply]
The system tracks closed connections in a ring buffer that you can query, but you also need access to the system account for that as well.
